src/Command/ScanForErrorMiddleware/NextInvocationScanner.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Expressive\Command\ScanForErrorMiddleware;
+
+class NextInvocationScanner
+{
+    const TEMPLATE = <<< 'EOT'
```

Comment:
I've just checked why single quotes here:
http://php.net/manual/en/language.types.string.php#language.types.string.syntax.nowdoc
I didn't know about it. Nice 👍